hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -263,7 +263,7 @@ private class Emit(
       mb.getArg[Array[RegionValueAggregator]](2)
     }
 
-    ir match {
+    (ir: @unchecked) match {
```

Comment:
I feel like this isn't the right answer. Shouldn't Emittable be an sub-interface of `IR`, and then we'll get useful compiler errors if we state that an IR is Emittable but it doesn't appear here?

Reply:
I agree this is a better design. I'd prefer not to do that here, since I view this PR as cleaning up compiler warnings from matches that were (for better or worse) intentionally providing no base case.